### PR TITLE
ci: fix lint command to support relocated APIs

### PR DESCRIPTION
**Pull request by stephenplusplus**
Fixes #2629

I can't reproduce the error locally, so this will be trial and error with CI on this PR.

### CODE REVIEW

**Comment by stephenplusplus** [comment]
Well, these tests passed, which I'm not sure means anything. I'll try an edit of a random file from a random package.

File: scripts/helpers.js [hunk 1]
```diff
@@ -60,6 +60,11 @@ function Module(name) {
   this.name = name;
   this.directory = path.join(ROOT_DIR, 'packages', name);
 
+  if (!test('-e', this.directory)) {
```
Comment (callmehiphop) [inline scripts/helpers.js]: I think this is the wrong spot, if I had a guess I would say you want to filter out deleted modules [here](https://github.com/GoogleCloudPlatform/google-cloud-node/blob/master/scripts/helpers.js#L135).

Comment (stephenplusplus) [reply scripts/helpers.js]: Thanks for that!

**Comment by stephenplusplus** [comment]
@callmehiphop PTAL!

**Pull request merged**